modules/nw-metallb-operator-custom-resources.adoc

```diff
@@ -20,13 +20,13 @@ An `IPAddressPool` requires a name.
 The documentation uses names like `doc-example`, `doc-example-reserved`, and `doc-example-ipv6`.
 An `IPAddressPool` assigns IP addresses from the pool.
 `L2Advertisement` and `BGPAdvertisement` custom resources enable the advertisement of a given IP from a given pool.
+You can assign IP addresses from an `IPAddressPool` to services and namespaces that you specify by using the `IPAddressPool` custom resource.
```

Comment:
Controller can assign IP addresses from  `IPAddressPool` to services in specific namespaces that are specified in the 'IPAddressPool' custom resource.

Reply:
hhmmm... I updated the sentence. Hopefully it's more specific now. I think any further details are better left to the example.